src/test/java/step5/domain/CarTest.java

```diff
@@ -0,0 +1,41 @@
+package step5.domain;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class CarTest {
+
+    private Car car;
+
+    @BeforeEach
+    void init() {
+        car = new Car("aaaa", 1);
+    }
+
+    @DisplayName("car 생성자 확인 - 정상 생성")
+    @Test
+    void carTest() {
+        Car first = new Car("aa", 1);
+        Car second = new Car("aa", 1);
+
+        assertEquals(first, second);
+    }
+
+    @DisplayName("car 이동 테스트 - 이동했을 경우")
+    @Test
+    void carMoveTest() {
+        car.move(4);
+        assertEquals(car.getPosition(), new Car("bbbb", 2).getPosition());
+    }
```

Comment:
다음과 같은 테스트 코드는 어떨까요? 😄 
```suggestion
    void carMoveTest() {
        car.move(4);
        
        assertEquals(car.getPosition(), new Position(2));
    }
```